docker/Dockerfile-bare

```diff
@@ -33,5 +33,6 @@ ENV PATH $JAVA_HOME/bin:/zap/:$PATH
 ENV ZAP_PATH /zap/zap.sh
 ENV HOME /home/zap/
 ENV ZAP_PORT 8080
+ENV IS_CONTAINERIZED true
```

Comment:
This image does not use the Python scripts, does no harm either.

Reply:
Ah right, good catch 👍